<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSelectionWindow</class>
 <widget class="QMainWindow" name="TableSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор таблицы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnPersonnel">
      <property name="text">
       <string>Персонал</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDepartment">
      <property name="text">
       <string>Отделы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPosition">
      <property name="text">
       <string>Должности</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAssignment">
      <property name="text">
       <string>Назначения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEducation">
      <property name="text">
       <string>Образование</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWorkExperience">
      <property name="text">
       <string>Опыт работы</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
